reece/javascript/lab02/hello-world.test.js

```diff
@@ -0,0 +1,10 @@
+// Reece Adams - ADV JS - 3/29/22 - lab 3: Hello Node - Version 2 //
+
+const {speak, capitalize} = require("./hello-world.js");
+const assert = require('assert');
+
+const name = "JEFF"
+speak(capitalize(name))
+
+assert.strictEqual(name, "Jeff");
```

Comment:
Tests should be written to test the return of a function. The way you have your tests structured here would be like:
```javascript
name = "Joe"
if ("Joe" == name)
```

instead we want to test the return of a function call to make sure it is returning the correct data
```javascript
assert.strictEqual("Jeff", capitalize("jeff"))
```
The return from `capitalize("jeff")` should equal "Jeff"